readthedocs/projects/tasks.py

```diff
@@ -229,6 +222,43 @@ def setup_vcs(self):
             raise BuildEnvironmentError('Failed to import project',
                                         status_code=404)
 
+    def get_bash_env(self):
+        """
+        Get bash environment variables used for all builder commands.
```

Comment:
`BuildCommand` defaults to `shell=False`, which should be what we're using for almost all commands. Naming/referencing `bash` here is misleading, these are just env variables as far as subprocess is concerned.